_frame.R

```diff
@@ -1,10 +1,11 @@
 text_to_frame <- function(text, file, width = 1920, height = 360,
   bg = "#F8F8F8", fg = "#404040", ...) {
 
-  par(..., bg = bg)
-  png(file = file, width = width, height = height, bg = bg,
+  ragg::agg_png(filename = file, width = width, height = height, bg = bg,
     pointsize = 12)
-  par(mar = c(0,0,0,0))
+  on.exit(dev.off())
+
+  par(bg = bg, mar = c(0,0,0,0), ...)
```

Comment:
I'm pretty sure setting `par()` before opening the graphics device didn't do anything.